src/recaptcha-wrapper.js

```diff
@@ -11,6 +11,9 @@ function getOptions() {
 function getURL() {
   const dynamicOptions = getOptions();
   const hostname = dynamicOptions.useRecaptchaNet ? "recaptcha.net" : "www.google.com";
+  if (dynamicOptions.enterprise) {
+    return `https://${hostname}/recaptcha/enterprise.js?onload=${callbackName}&render=explicit`;
```

Comment:
is *recaptcha.net* a valid domain for enterprise? I've never seen this myself. Might be safer to assume not unless you can confirm

Reply:
@moonkid196 I would think both options would work because recaptcha.net is really just a redirect on the other domain to avoid blocks on google's domain